### DISCUSSION TITLE
WIN11 Professional workstation 23H2 22631.3155 Cannot run normally after installation is complete

### DISCUSSION THREAD

**Discussion by INTHECN** [valinet/ExplorerPatcher#3042] [Q&A]
After the installation is complete, the program starts repeatedly until prompted to uninstall.Hope this issue can be fixed soon.
Salute above.

**Comment by pyrates999** [comment]
What do you mean the program starts repeatedly?  What program?

**Reply by INTHECN** [reply]
It's literally rebooting over and over again, and after three repetitions it stops being enabled, prompting incompatibility.

**Reply by pyrates999** [reply]
Your windows build indicates you only installed updates from February: https://support.microsoft.com/en-us/topic/february-13-2024-kb5034765-os-builds-22621-3155-and-22631-3155-023d9141-a5b0-470a-b0f5-2ee3b48f48ce

Please install the latest windows update available from April: https://support.microsoft.com/en-us/topic/april-9-2024-kb5036893-os-builds-22621-3447-and-22631-3447-a674a67b-85f5-4a40-8d74-5f8af8ead5bb

I would advise not installing preview updates though.

In the settings app, untick "Get the latest updates as soon as available" so that preview updates are not automatically installed.

All preview updates are then shown as download and install and are never automatically installed if you have the above unticked.

Preview updates are also not fully tested by Microsoft and thus could have bugs.

You should only be installing preview updates if you are a developer or are OK with running pre-release versions of EP.
